### PR TITLE
ZD-798 Add updateJob method

### PR DESCRIPTION
**Pull request by vfisyuk-smartling**

### CODE REVIEW

File: api/jobs/params/update-job-parameters.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { BaseParameters } from "../../parameters/index";
+import { SmartlingException } from "../../exception/index";
+
+interface CustomFieldBasicRequest {
+    fieldUid: string,
+    fieldValue: string
+}
+
+export { CustomFieldBasicRequest };
```
Comment (vsolovei-smartling) [inline api/jobs/params/update-job-parameters.ts:9]: Looks like this export is not needed. Maybe, the interface could be inlined as well.

**Review by vsolovei-smartling — APPROVED** [review]

File: api/jobs/params/update-job-parameters.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+export class UpdateJobParameters extends BaseParameters {
```
Comment (PavelLoparev) [inline api/jobs/params/update-job-parameters.ts:11]: Maybe extend `CreateJobParameters` from `UpdateJobParameters` to avoid copy-pasting of same methods?

**Review by PavelLoparev — APPROVED** [review]

File: api/jobs/params/update-job-parameters.ts (hunk 1 above)
Comment (vfisyuk-smartling) [reply api/jobs/params/update-job-parameters.ts]: Removed export.

**Review by PavelLoparev — CHANGES_REQUESTED** [review]
Please do not forget to update version in package* files

**Comment by vfisyuk-smartling** [comment]
> Please do not forget to update version in package* files

Sure, I was going to do that after merging.

**Comment by PavelLoparev** [comment]
> > Please do not forget to update version in package* files
> 
> Sure, I was going to do that after merging.

But why after? Maybe I'm missing something, but I always update those files in branch directly

File: api/jobs/params/create-update-job-base-parameters.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        this.set("callbackUrl", callbackUrl);
+
+        return this as unknown as T;
+    }
+
+    setCallbackMethod(callbackMethod: string | null): T {
+        this.set("callbackMethod", callbackMethod);
+
+        return this as unknown as T;
+    }
```
Comment (PavelLoparev) [inline api/jobs/params/create-update-job-base-parameters.ts:60]: Shouldn't it be enum with only `POST | GET` methods?

**Review by PavelLoparev — APPROVED** [review]

File: api/jobs/params/create-update-job-base-parameters.ts (hunk 3 above)
Comment (vfisyuk-smartling) [reply api/jobs/params/create-update-job-base-parameters.ts]: Makes sense, have added.

**Review by PavelLoparev — APPROVED** [review]
Please rebase from fresh code master branch

**Comment by vfisyuk-smartling** [comment]
> Please rebase from fresh master branch

Have already merged from master.

**Review by dimitrystd — COMMENTED** [review]
Thx. I didn't expect so many changes.

**Pull request merged**